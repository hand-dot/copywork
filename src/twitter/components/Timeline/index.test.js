import React from 'react';
import ReactDOM from 'react-dom';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import Index from './';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Index />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders correctly', () => {
  const wrapper = shallow(<Index />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
