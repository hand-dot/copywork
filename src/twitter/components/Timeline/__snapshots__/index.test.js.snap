// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<WithStyles(ForwardRef(Paper))>
  <div
    style={
      Object {
        "display": "flex",
        "padding": 10,
      }
    }
  >
    <WithStyles(ForwardRef(Avatar))
      alt="Remy Sharp"
      src="https://next.material-ui.com/static/images/avatar/1.jpg"
    />
    <input
      placeholder="いまどうしてる？"
      style={
        Object {
          "flexGrow": 1,
        }
      }
    />
  </div>
  <div
    style={
      Object {
        "display": "flex",
        "padding": 10,
      }
    }
  >
    <WithStyles(ForwardRef(Avatar))
      alt="Remy Sharp"
      src="https://next.material-ui.com/static/images/avatar/1.jpg"
    />
    <WithStyles(ForwardRef(Typography))
      component="h2"
      gutterBottom={true}
      variant="h5"
    >
      Lizard
    </WithStyles(ForwardRef(Typography))>
  </div>
  <div
    style={
      Object {
        "display": "flex",
        "padding": 10,
      }
    }
  >
    <WithStyles(ForwardRef(Avatar))
      alt="Remy Sharp"
      src="https://next.material-ui.com/static/images/avatar/1.jpg"
    />
    <WithStyles(ForwardRef(Typography))
      component="h2"
      gutterBottom={true}
      variant="h5"
    >
      Lizard
    </WithStyles(ForwardRef(Typography))>
  </div>
  <div
    style={
      Object {
        "display": "flex",
        "padding": 10,
      }
    }
  >
    <WithStyles(ForwardRef(Avatar))
      alt="Remy Sharp"
      src="https://next.material-ui.com/static/images/avatar/1.jpg"
    />
    <WithStyles(ForwardRef(Typography))
      component="h2"
      gutterBottom={true}
      variant="h5"
    >
      Lizard
    </WithStyles(ForwardRef(Typography))>
  </div>
  <div
    style={
      Object {
        "display": "flex",
        "padding": 10,
      }
    }
  >
    <WithStyles(ForwardRef(Avatar))
      alt="Remy Sharp"
      src="https://next.material-ui.com/static/images/avatar/1.jpg"
    />
    <WithStyles(ForwardRef(Typography))
      component="h2"
      gutterBottom={true}
      variant="h5"
    >
      Lizard
    </WithStyles(ForwardRef(Typography))>
  </div>
  <div
    style={
      Object {
        "display": "flex",
        "padding": 10,
      }
    }
  >
    <WithStyles(ForwardRef(Avatar))
      alt="Remy Sharp"
      src="https://next.material-ui.com/static/images/avatar/1.jpg"
    />
    <WithStyles(ForwardRef(Typography))
      component="h2"
      gutterBottom={true}
      variant="h5"
    >
      Lizard
    </WithStyles(ForwardRef(Typography))>
  </div>
  <div
    style={
      Object {
        "display": "flex",
        "padding": 10,
      }
    }
  >
    <WithStyles(ForwardRef(Avatar))
      alt="Remy Sharp"
      src="https://next.material-ui.com/static/images/avatar/1.jpg"
    />
    <WithStyles(ForwardRef(Typography))
      component="h2"
      gutterBottom={true}
      variant="h5"
    >
      Lizard
    </WithStyles(ForwardRef(Typography))>
  </div>
</WithStyles(ForwardRef(Paper))>
`;
